<ui version="4.0" >
 <class>widget_changemap</class>
 <widget class="QWidget" name="widget_changemap" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Widget_changemap</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label_12" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Game modes</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item rowspan="5" row="0" column="1" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2" >
    <widget class="QLabel" name="label_14" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Map details</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QComboBox" name="gamerulesList" />
   </item>
   <item rowspan="3" row="1" column="2" colspan="2" >
    <widget class="QLabel" name="mapScreenshot" >
     <property name="minimumSize" >
      <size>
       <width>478</width>
       <height>270</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>478</width>
       <height>270</height>
      </size>
     </property>
     <property name="text" >
      <string>map screenshot</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_13" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Maps</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="3" column="0" >
    <widget class="QListWidget" name="mapList" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2" >
    <widget class="QLabel" name="mapDescription" >
     <property name="text" >
      <string>map description</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <spacer name="horizontalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>377</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3" >
    <widget class="QPushButton" name="mapSend_bt" >
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text" >
      <string>Change map</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
